### ISSUE TITLE
Spring Reverb 1.0.1 Polyphony?

### ISSUE THREAD

**Issue by pgatt** [martin-lueders/Befaco#5]
Hi Martin, on 1.0.1 Spring Reverb is polyphonic in but not out - is this the expected behaviour?

Thanks.

**Comment by martin-lueders** [comment]
Yes, I did not see the point in a polyphonic reverb, as it would usually be used as effect. The input then has the standard behaviour of mixing the channels. Do you think a polyphonic reverb would make sense?
